openedx/core/djangoapps/catalog/management/commands/sync_course_runs.py

```diff
@@ -0,0 +1,55 @@
+"""
+Sync course runs from catalog service.
+"""
+import logging
+
+from django.core.management.base import BaseCommand
+from opaque_keys.edx.keys import CourseKey
+
+from catalog.utils import get_catalog_course_runs
+from content.course_overviews.models import CourseOverview
+
+logger = logging.getLogger(__name__)
+
+
+def fetch_catalog_course_runs():
+    """
+    Fetch catalog course runs from catalog service for all course overview records.
+    """
+    course_overviews = CourseOverview.objects.all()
```

Comment:
@nasthagiri 1 and 2 are not true. See [this line](https://github.com/edx/course-discovery/blob/master/course_discovery/apps/course_metadata/data_loaders/api.py#L123) in the `CoursesApiDataLoader`. The catalog includes all courses available on the LMS, even if they're not fronted by a separate marketing site. CCX courses are also ingested.